src/test/java/net/helpscout/api/ApiClientTest.java

```diff
@@ -64,6 +77,33 @@ public void shouldReturnDetailedErrorMessageForBadRequest() {
         }
     }
     
+	@Test
+	public void shouldRetrieveSpecificCustomer() throws ApiException {
+		givenThat(get(urlEqualTo("/v1/customers/60984612.json"))
+				.willReturn(aResponse().withStatus(HTTP_OK)
+				.withBody(getResponse("customer"))));
```

Comment:
For the life of me I couldn't get it to work, didn't want to waste too much time on it so just copied the way wiremock is used in other projects.